qlty-cloud/src/auth/credentials.rs

```diff
@@ -0,0 +1,48 @@
+use anyhow::{Context, Result};
+use keyring::Entry;
+use std::sync::{Arc, LazyLock, Mutex};
+
+pub const SERVICE: &str = "qlty-cli";
+pub const DEFAULT_USER: &str = "default";
+
+static ENTRY: LazyLock<Mutex<Option<Arc<Entry>>>> = LazyLock::new(|| Mutex::new(None));
```

Comment:
Do we need both LazyLock and Mutex?

Reply:
> Do we need both LazyLock and Mutex?

Yes because LazyLock is for initialization, not updates after init. I believe we could use lazy_static to simplify the line, but you'd still need the mutex.

Reply:
Got it, thanks. We use lazy_static in a few other places